apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx

```diff
@@ -143,8 +143,8 @@ const LanguageInformationSection = ({
               <span data-h2-display="base(block)">
                 {intl.formatMessage({
                   defaultMessage:
-                    "Second language level (Comprehension, Written, Verbal):",
-                  id: "D7Qb41",
+                    "Second language level (Reading, Writing, Oral interaction):",
```

Comment:
Change to **Second language level (reading, writing, oral interaction)** and remove the colon from the string. Add the dividing colon message instead:

`{intl.formatMessage(commonMessages.dividingColon)}`